src/main/java/org/sonatype/nexus/ci/nxrm/ComponentUploader.groovy

```diff
@@ -112,7 +112,7 @@ class ComponentUploader
     logger.println('Successfully Uploaded Maven Assets')
   }
 
-  RepositoryManagerClient getRepositoryManagerClient(final NxrmConfiguration nexusConfiguration)
+  RepositoryManagerV2Client getRepositoryManagerClient(final NxrmConfiguration nexusConfiguration)
```

Comment:
Just curious, why does this need to change?

Reply:
> Just curious, why does this need to change?

`RepositoryManagerClient` doesn't exist in the new version.  I had to upgrade the nexus-java-api to get the new verifyOrCreate, which is needed for this story. The upgrade also brought in a bunch of other things that needed to be changed.  So I tried to fix the problem.  I wasn't sure about my changes.  If I should use v2 or v3.  Do I have to have new logic to support this?  So I brought it up in the integration meeting.  Justin suggested I do not make these changes but have Kristofer or some else on a different team do it and I remove these changes from this PR.  This will cause an issue for this PR.  Because the old RepositoryManagerClient  does not exist in a new version and my code will not compile without this being fixed.  I may have to table this PR and wait for the changes.  I really don't know how to handle this.

Reply:
> Just curious, why does this need to change?

Update, this code will be removed from my PR after Kristofer makes the changes.  Once he does, I will rebase of them.

Reply:
These changes will be made in a separate PR by @wrightkris and this PR can rebase against those (or master when they are merged as they are trivial).

Reply:
https://github.com/sonatype/nexus-platform-plugin/pull/2

Merged the minor updates necessary to work against the released 3.0 api.